app/views/questions/show.html.erb

```diff
@@ -1,13 +1,15 @@
-<p id="notice"><%= notice %></p>
+<div class="container--small content-padding">
+  <p id="notice"><%= notice %></p>
 
-<% I18n.available_locales.each do |locale| %>
-  <h1><%= locale %></h1>
-  <% I18n.locale = locale %>
-  <p>
-    <strong>Text:</strong>
-    <%= @question.text %>
-  </p>
-<% end %>
+  <% I18n.available_locales.each do |locale| %>
+    <h1><%= locale %></h1>
```

Comment:
Same again, do we want to use `Globalize.with_locale(locale)` here?

Reply:
This is in develop already I think :)